test/services/videoconference/index.test.js

```diff
@@ -0,0 +1,268 @@
+const { expect } = require('chai');
+const rp = require('request-promise-native');
+const app = require('../../../src/app');
+const testObjects = require('../helpers/testObjects')(app);
+
+const { generateRequestParamsFromUser } = require('../helpers/services/login')(app);
+
+const createService = app.service('videoconference');
+const getService = app.service('videoconference/:scopeName');
+
+const { VIDEOCONFERENCE } = require('../../../src/services/school/model').SCHOOL_FEATURES;
+
+const getServiceRespondsDependingOnUserPermissions = (testData) => {
+	it('test get with start or join permission works after creation', async () => {
+		// expect creation finished like above...
+		const teacherResponse = getService
+			.get(testData.serviceParams.scopeId, {
+				route: { scopeName: testData.serviceParams.scopeName },
+				...testData.teacherRequestAuthentication,
+			});
+		const substitutionTeacherResponse = getService
+			.get(testData.serviceParams.scopeId, {
+				route: { scopeName: testData.serviceParams.scopeName },
+				...testData.substitutionTeacherRequestAuthentication,
+			});
+		const courseStudentResponse = getService
+			.get(testData.serviceParams.scopeId, {
+				route: { scopeName: testData.serviceParams.scopeName },
+				...testData.studentRequestAuthentication,
+			});
+		let successCounter = 0;
+		await Promise.all([teacherResponse, substitutionTeacherResponse, courseStudentResponse])
+			.then(([...serviceResponses]) => Promise.all(serviceResponses.map((serviceResponse) => {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected to return a value in arrow function.](https://app.codacy.com/manual/schul-cloud/schulcloud-server/pullRequest?prid=4819756)